build/Program.cs

```diff
@@ -41,55 +39,49 @@ static void Main(string[] args)
 
             Target(Targets.Pack, DependsOn(Targets.Build), () =>
             {
-                var project = Directory.GetFiles("./src", "*.csproj", SearchOption.TopDirectoryOnly).First();
```

Comment:
Since the value of `.First()` has an absolute expectation, may as well hardcode it.